source/docs/articles/organizations/change-management.md

```diff
@@ -6,46 +6,59 @@ category:
   - developing
 keywords: roles, role, permissions, permission, access, privileges, privilege, team members, developer role, developer, organization management, manage organization, change management, team management, manage team, manage team access  
 ---
-Change Management is a feature of Organizations that enables role-based permissions for users on the Organization. Sites which list the Organization as a supporting Organization are accessible to privileged members of the Organization. The roles exist to restrict who can deploy and manage other users in the Organization or sites it works on.
+Change Management is an Organization-level feature that enables role-based permissions for users in the Organization. It is automatically enabled for all [Pantheon Partners](/partners), and is available as an add-on for Enterprise Organizations. Partner users can access all sites associated with the organization, with access restricted based on the user's role in that organization. These roles exist to restrict who can deploy and manage other users in the Organization or sites it works on.
 
 ## Features
-* Organizations with Change Management can add people as Administrators, Team Members, and Developers to the Organization. Admins have all privileges; Team Members are a trusted role that can make changes that affect the Test and Live environments; and Developers can only work on Development environments.
-* When an Organization is added to a site as a supporting Organization (e.g. Agencies), the members and roles of the Organization are inherited for working on the site. All Organization members can access all sites associated with the organization.
-* Agencies added to Sites by subscription owners are able to use Change Management internally, but the subscription owners are not—they can only add full team members.
+* Enterprise organization administrators and site team members can add Users or Supporting Organizations to sites owned by Organization with the Developer or Team Member roles. 
+* Partner Organizations can have users with a role: Administrator, Team Member, or Developer. 
+* When an Organization is added to a site as a Supporting Organization (e.g. Agencies), the members and roles of the Organization are inherited for working on the site, unless the site is owned by an Enterprise Organization with change management enabled. In this case, the site-level role is respected for all users in the Supporting Organization.
+* Agencies added to sites by subscription owners are able to use Change Management internally.
```

Comment:
I think this line should be removed because it's not clear that "internally" means "On the organization Dashboard"